foreman/data_refinery_foreman/foreman/management/commands/update_experiment_metadata.py

```diff
@@ -0,0 +1,90 @@
+from django.core.management.base import BaseCommand
+import GEOparse
+import sys
+import time
+
+from data_refinery_common.models import Experiment
+from data_refinery_common.logging import get_and_configure_logger
+from data_refinery_foreman.foreman.performant_pagination.pagination import PerformantPaginator
+from data_refinery_foreman.surveyor import utils
+from data_refinery_foreman.surveyor.sra import SraSurveyor
+from data_refinery_foreman.surveyor.geo import GeoSurveyor
+from data_refinery_foreman.surveyor.array_express import ArrayExpressSurveyor, EXPERIMENTS_URL
+
+logger = get_and_configure_logger(__name__)
+
+PAGE_SIZE = 2000
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--source-database",
+            type=str,
+            help=("The name of a source database, such as ARRAY_EXPRESS, GEO, or SRA."
+                  "All samples from this source database will have their metadata refreshed."))
+
+    def handle(self, *args, **options):
+        """Refreshes the metadata for RNA-Seq samples
```

Comment:
This docstring and the helpstring above both say samples, but this doesn't update the samples, it does the experiments. This docstring also specifies RNA-Seq but I think it'll work on all experiments.

Reply:
My bad, I originally copied the refresh samples file for this, and I must not have changed all of the comments